WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -262,10 +263,9 @@ private void resetEditTextContent(EditText editText) {
         }
     }
 
-    private String loadDefaultRole() {
-        int roleRes = isPrivateSite() ? R.array.invite_roles_private : R.array.invite_roles_public;
-        final String[] roles = getResources().getStringArray(roleRes);
-        return roles[0];
+    private Role loadDefaultRole() {
```

Comment:
Hey @oguzkocer , thanks for the ping!

I chose `loadDefaultRole` to hint the caller that this is not be a very cheap operation (it always loads from the Resources, and it loads the whole string array only to get its first item) and should not be treated like a simple "getter". So, the caller better cache the returned value if needed more than once.

Not a super important reason as the resource loading is not that heavy in this case so, I think we can keep the new name. +1